pages/common/tlmgr-check.md

```diff
@@ -0,0 +1,28 @@
+# tlmgr check
+
+> Check the consistency of a TeX Live installation.
+> More information: <https://www.tug.org/texlive/tlmgr.html>.
+
+- Check the consistency of the whole TeX Live installation:
```

Comment:
```suggestion
- Check the consistency of the whole installation:
```

Reply:
Why do you think it should be removed? The sentence is quite short and more explicit is always better.

Reply:
> more explicit is always better

Do we have any rule that states that? Because I disagree a lot. There are many horrible ways to make this a lot more explicit about everything. The description explicitly mentions that this operates on a TeX Live installation, so this isn't just explicit, it is redundant.

Reply:
We do have a rule which states that examples should not depend on the others and can be read on it's own. @CleanMachine1  / @navarroaxel wdyt?

Reply:
In this case I think mention `TeX Live` is better.

Reply:
I think keeping this is best, because it's more explicit. Otherwise, one would question which executables it checks.

Reply:
Its fine to me. Same opinion as others thoughts.